### PR TITLE
UI: Is it you? BULMA?

### PR DESCRIPTION
**Pull request by frozeeen**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,16 +3,34 @@
 <head>
     <meta charset="UTF-8">
     <title>phzipsearch</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
```
Comment (kulotsystems) [inline index.html]: Thanks @frozeeen.

But sorry, we don't do CDN in this company. XD

**Review by kulotsystems — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (kulotsystems) [reply index.html]: To keep it simple, please just download [Bulma](https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css) to `/public/css` and link it from there.

File: index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <title>phzipsearch</title>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
+    <link rel="stylesheet" type="text/css" href="src/bulma.min.css">
```
Comment (kulotsystems) [inline index.html]: Let's keep first-party contents inside the `src` folder.
Please move [Bulma](https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css) to `/public/css`.

Then you can link it as `href="/css/bulma.min.css"`.

**Review by kulotsystems — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -3,16 +3,34 @@
 <head>
     <meta charset="UTF-8">
     <title>phzipsearch</title>
+    <link rel="stylesheet" type="text/css" href="/css/bulma.min.css">
```
Comment (kulotsystems) [inline index.html:6]: LGTM

**Review by kulotsystems — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (frozeeen) [reply index.html:6]: Yey, thank you CEO @kulotsystems! 😙👌

**Pull request merged**